File: server/core/model/flow_version.go
package model

type FlowVersion struct {
	BaseModel
	FlowID            int64  `gorm:"column:flow_id" json:"flowId"`
	FlowVersion       string `gorm:"column:flow_version;size:8" json:"flowVersion"`
	FlowVersionStatus int    `gorm:"column:flow_version_status;default:0" json:"flowVersionStatus"`
	FlowVersionRemark string `gorm:"column:flow_version_remark;size:200" json:"flowVersionRemark,omitempty"`
	FlowContent       string `gorm:"column:flow_content;type:mediumtext" json:"flowContent,omitempty"`
	Inputs            string `gorm:"column:inputs;type:text" json:"inputs,omitempty"`
	Outputs           string `gorm:"column:outputs;type:text" json:"outputs,omitempty"`
	Variables         string `gorm:"column:variables;type:text" json:"variables,omitempty"`
}

func (FlowVersion) TableName() string { return "t_flow_version" }

File: commom/src/main/java/com/javaoffers/nexus/common/model/FlowVersion.java
package com.javaoffers.nexus.common.model;

import com.javaoffers.brief.modelhelper.anno.BaseModel;
import com.javaoffers.brief.modelhelper.anno.BaseUnique;
import com.javaoffers.brief.modelhelper.anno.ColName;
import com.javaoffers.brief.modelhelper.anno.derive.flag.IsDel;
import lombok.Data;

import java.util.Date;

@BaseModel("t_flow_version")
@Data
public class FlowVersion {
    @BaseUnique
    private Long id;
    private IsDel deleted;
    private Date createdAt;
    private Long createdBy;
    private Date updatedAt;
    private Long updatedBy;

    @ColName(value = "id", excludeColAll = true)
    private Long countId;

    private Long flowId;
    private String flowVersion;
    private Integer flowVersionStatus;
    private String flowVersionRemark;
    private String flowContent;
    private String inputs;
    private String outputs;
    private String variables;
}
